moveit_plugins/moveit_simple_controller_manager/src/follow_joint_trajectory_controller_handle.cpp

```diff
@@ -48,18 +48,14 @@ bool FollowJointTrajectoryControllerHandle::sendTrajectory(const moveit_msgs::ms
   if (!controller_action_client_)
     return false;
 
-  if (!trajectory.multi_dof_joint_trajectory.points.empty())
-  {
-    RCLCPP_WARN_STREAM(LOGGER, name_ << " cannot execute multi-dof trajectories.");
-  }
-
   if (done_)
     RCLCPP_INFO_STREAM(LOGGER, "sending trajectory to " << name_);
   else
     RCLCPP_INFO_STREAM(LOGGER, "sending continuation for the currently executed trajectory to " << name_);
 
   control_msgs::action::FollowJointTrajectory::Goal goal = goal_template_;
   goal.trajectory = trajectory.joint_trajectory;
+  goal.multi_dof_trajectory = trajectory.multi_dof_joint_trajectory;
```

Comment:
so... this doesn't seem to exist?

Reply:
This only exists in galactic

Reply:
ok, so we need to hold this for now until we branch off Foxy